shared/nav.native.js

```diff
@@ -82,38 +72,66 @@ function renderMainStackRoute (route) {
   )
 }
 
-function MainNavStack (props: Props) {
+function cardStackShim (props: Props) {
```

Comment:
This is effectively a stateless component so it should be capitalized and treated like a component